doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+article.
+
+To do this, you will need to change the ``tarantool/cartridge`` chart. You
+can find it
+`here <https://github.com/tarantool/tarantool-operator/tree/master/examples/kv/helm-chart>`_.
+Add a new container with ``nginx`` to the `ReplicasetTemplate which can be
+found in the ``templates/deployment.yaml`` file.
+
+.. code:: yaml
+
+   containers:
+   - name: "pim-storage"
+     image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag }}"
+     ...
+   - name: "nginx-container"
+     image: "nginx"
+     volumeMounts:
+       - name: "www"
+         mountPath: "/data"
+
+.. NOTE::
+
+   It is important to describe additional containers strictly
+   after the pim-storage container. Otherwise, problems may occur when
+   updating the version of the application.
+
+   By default, Tarantool Kubernetes operator
+   chooses the first one in the list as the application container.
+
+Now, let's start the installation specifying the path to the directory
+with the customized chart.
+
+.. code-block:: console
+
+   $ helm install -f values.yaml test-app tarantool-operator/examples/kv/helm-chart/ --namespace tarantool
+   ---
+   NAME: test-app
+   LAST DEPLOYED: Wed Sep 30 11:25:12 2020
+   NAMESPACE: tarantool
+   STATUS: deployed
+   REVISION: 1
+
+If everything goes well, it will be visible in the pod list:
+
+.. code-block:: console
+
+   $ kubectl -n tarantool get pods
+   ---
+   NAME                                  READY   STATUS    RESTARTS   AGE
+   routers-0-0                           2/2     Running   0          113s
+   routers-1-0                           2/2     Running   0          113s
+   storages-0-0                          2/2     Running   0          113s
+   tarantool-operator-7dc948c89b-fnq28   1/1     Running   0          30m
+
+``READY 2/2`` means that 2 containers are ready inside the pod.
+
+.. _cartridge_kubernetes_installation_on_the_internal_network:
+
+--------------------------------------------------------------------------------
+Installation in the internal network
+--------------------------------------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Delivery of tools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We need to bring the ``tarantool-cartridge`` and ``tarantool-operator``
+charts and the image of your application inside the internal network.
+
+You can download the charts from the following links:
+
+* `tarantool-operator v0.0.8 <https://github.com/tarantool/tarantool-operator/releases/download/tarantool-operator-0.0.8/tarantool-operator-0.0.8.tgz>`_
+* `cartridge v0.0.8 <https://github.com/tarantool/tarantool-operator/releases/download/cartridge-0.0.8/cartridge-0.0.8.tgz>`_.
+
+Next, you need to pack the docker image with ``tarantool-operator``.
+First, let's pull the required version from ``docker.hub``:
```

Comment:
Is `docker.hub` some code entity or are we speaking about the [Docker Hub](https://hub.docker.com/)?